### PR TITLE
fix: support of activation non default podman machines

### LINKED ISSUE #137: Subscription activation does not work for non default podman VM
1. Create VM named 'machine1'
2. Start it and replay to no to request to make it default
3. Try to login to Red Hat SSO 

(3) fails because only machine1 is running and is is not default. The command `podman machine ssh` will fail because it assumes there is default machine is running.

### PR DESCRIPTION
**Pull request by dgolovin**
Fix #137.

### CODE REVIEW

File: src/subscription.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    [
+      'api.iam.registry_service_accounts', //scope that gives access to hydra service accounts API
+      'api.console',
+    ], // scope that gives access to console.redhat.com APIs
+    { createIfNone }
+  );
+}
+
+// TODO: add listRegistries to registry API to allow search by
+// registry URL
```
Comment (benoitf) [inline src/subscription.ts:22]: listRegistries might be async so I I don't know if your function is synchronous if it'll work as expected (you'll need some refactoring)

Comment (benoitf) [reply src/subscription.ts:22]: I was wondering if you can use `onDidRegisterRegistry` method to collect registries ?

Comment (benoitf) [reply src/subscription.ts:22]: it should notify with all podman registries

Comment (dgolovin) [reply src/subscription.ts:22]: Those are all good options with pros and cons. Let's take care of it as follow up issue.

**Review by benoitf — APPROVED** [review]

**Pull request merged**